pkg/sfu/buffer/rtpstats.go

```diff
@@ -66,24 +66,24 @@ func (d driftResult) String() string {
 
 type RTPFlowState struct {
 	HasLoss            bool
-	LossStartInclusive uint32
-	LossEndExclusive   uint32
+	LossStartInclusive uint64
+	LossEndExclusive   uint64
 
 	IsOutOfOrder bool
 
-	ExtSequenceNumber uint32
+	ExtSequenceNumber uint64
 	ExtTimestamp      uint64
 }
 
 type IntervalStats struct {
-	packets            uint32
+	packets            uint64
 	bytes              uint64
 	headerBytes        uint64
-	packetsPadding     uint32
+	packetsPadding     uint64
 	bytesPadding       uint64
 	headerBytesPadding uint64
-	packetsLost        uint32
-	packetsOutOfOrder  uint32
+	packetsLost        uint64
+	packetsOutOfOrder  uint64
```

Comment:
some of these can be `uint32` (actually they are `uint32` in protocol/analytics stats), but just keeping them all uint64 internally and casting them when returning values as necessary.